app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectTypeRow/defectTypeRow.jsx

```diff
@@ -31,33 +31,28 @@ import { CopyToClipboard } from 'react-copy-to-clipboard';
 import { showModalAction } from 'controllers/modal';
 import { deleteDefectTypeAction, updateDefectTypeAction } from 'controllers/project';
 import { PROJECT_SETTINGS_DEFECT_TYPES_EVENTS } from 'analyticsEvents/projectSettingsPageEvents';
+import { NOTIFICATION_TYPES, showNotification } from 'controllers/notification';
 import { defectTypeShape } from '../defectTypeShape';
 import styles from './defectTypeRow.scss';
 
 const cx = classNames.bind(styles);
 
-const DefectLocatorPopoverContent = ({ locator }) => {
-  const { trackEvent } = useTracking();
-
-  return (
-    <div className={cx('defect-locator-popover')}>
-      Locator
-      <span className={cx('locator')}>
-        {locator}
-        <CopyToClipboard
-          text={locator}
-          onCopy={() => trackEvent(PROJECT_SETTINGS_DEFECT_TYPES_EVENTS.CLICK_COPY_ID_LOCATOR_ICON)}
-        >
-          <i className={cx('icon', 'copy-button')} data-automation-id="defectTypeCopyLocatorIcon">
-            {Parser(CopyIcon)}
-          </i>
-        </CopyToClipboard>
-      </span>
-    </div>
-  );
-};
+const DefectLocatorPopoverContent = ({ locator, onCopy }) => (
+  <div className={cx('defect-locator-popover')}>
+    Locator
+    <span className={cx('locator')}>
+      {locator}
+      <CopyToClipboard text={locator} onCopy={onCopy}>
+        <i className={cx('icon', 'copy-button')} data-automation-id="defectTypeCopyLocatorIcon">
+          {Parser(CopyIcon)}
+        </i>
+      </CopyToClipboard>
+    </span>
+  </div>
+);
 DefectLocatorPopoverContent.propTypes = {
   locator: PropTypes.string.isRequired,
+  onCopy: PropTypes.func,
```

Comment:
Could you please provide the default values for optional properties?

Reply:
Done.